### DISCUSSION TITLE
A guide or wrapper for projects using CSharp

### DISCUSSION THREAD

**Discussion by emperorstarfinder** [NVIDIA-Omniverse/PhysX#267] [Ideas]
I am not sure if there is a guide floating around or a wrapper for projects built in CSharp that properly utilizes the PhysX SDK.  If not, then is there a way to provide a guide to creating the necessary wrappers?

**Comment by alonfaraj** [comment]
@emperorstarfinder 
I didn't try it myself but came across this project. Might be a good start. 

https://github.com/Cysharp/MagicPhysX
